package.json

```diff
@@ -37,7 +37,7 @@
     "eslint-config-prettier": "^8.1.0",
     "eslint-plugin-import": "^2.22.1",
     "eslint-plugin-jest": "^24.3.4",
-    "eslint-plugin-jsdoc": "^36.1.0",
+    "eslint-plugin-jsdoc": "^39.2.9",
```

Comment:
This is the minimum version of the package that's compatible with Node 18. They make no note of this change in their changelog.